package-overrides/github/angular-ui/bootstrap-bower@0.14.2.json

```diff
@@ -0,0 +1,8 @@
+{
+  "registry": "jspm",
```

Comment:
Unfortunately this will break in jspm 0.17 as it will try to read the peerDependencies which are a version range format that is not supported in jspm (currently).

I'd suggest adding a `dependencies: { angular: '*' }` as well here, or leaving this property out entirely to ignore the dependencies completely.